modules/gapi/src/backends/cpu/gcpuimgproc.cpp

```diff
@@ -303,6 +328,25 @@ cv::gapi::GKernelPackage cv::gapi::imgproc::cpu::kernels()
         , GCPUBGR2Gray
         , GCPURGB2Gray
         , GCPURGB2GrayCustom
+        , GCPUBayerGR2RGB
+        , GCPURGB2HSV
+        , GCPURGB2YUV422
         >();
     return pkg;
 }
+
+void cv::gapi::convertRGB2YUV422(const cv::Mat& in, cv::Mat& out)
+{
+    if (out.size() != in.size())
+    {
+        out.create(in.size(), CV_8UC2);
+    }
+
+    for (int i = 0; i < in.rows; ++i)
+    {
+        const uchar* in_line_p  = in.ptr<uchar>(i);
+        uchar* out_line_p = out.ptr<uchar>(i);
+        // FIXME: scalar and vectorized version are not bitwise due to round (~0.1%)
+        cv::gapi::fluid::run_rgb2yuv422_impl(out_line_p, in_line_p, in.cols);
+    }
+}
```

Comment:
I believe you just can put this code block into the OCV kernel body and drop this function.

Reply:
Fixed